android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -273,6 +274,20 @@ public void onClick(View v) {
     return mainLayout;
   }
 
+
+  public static void setIsChiral(KeyboardType keyboardType) {
+    KMManager.getKeyboard(keyboardType).loadUrl("javascript:setIsChiral()");
+  }
+
+  public static void checkIsChiral(KeyboardType keyboardType) {
```

Comment:
Do `setIsChiral` and `checkIsChiral` do the same thing?

Reply:
oh right. I refactored `setIsChiral` to `checkIsChiral` and forgot to remove the old one.